lib/src/divider/types.ts

```diff
@@ -0,0 +1,22 @@
+export type Width = "fixed" | "30%" | "50%" | "80%" | "Full-Width";
```

Comment:
We should use fullWidth or fullLength

Reply:
It has been decided that length/width is no longer part of the component.